### DISCUSSION TITLE
Using image/icon as label instead of text

### DISCUSSION THREAD

**Discussion by emmajane1313** [rainbow-me/rainbowkit#643] [Q&A] [answered]
Is there a way to customise the label of a button to display an image/icon instead of text?

**Comment by nickbytes** [comment, marked as answer]
@emmajane1313 yep – you can create a Custom Connect Button that could be an image! Docs on creating a Custom Connect Button are here: https://www.rainbowkit.com/docs/custom-connect-button

You could replace the `button` element here with any markup you'd like

<img width="690" alt="image" src="https://user-images.githubusercontent.com/3009331/184050636-16db8c80-912c-47a2-a1d7-0c29dcb985be.png">
